src/main/java/org/thoughtcrime/securesms/contacts/ContactSelectionListAdapter.java

```diff
@@ -268,6 +268,8 @@ public void onBindViewHolder(@NonNull ViewHolder viewHolder, int i) {
       itemMultiSelect = false; // the item creates a new contact in the list that will be selected instead
     } else if (id == DcContact.DC_CONTACT_ID_NEW_GROUP) {
       name = context.getString(R.string.menu_new_group);
+    } else if (id == DcContact.DC_CONTACT_ID_NEW_UNENCRYPTED_GROUP) {
+      name = context.getString(R.string.new_email);
```

Comment:
I think that "New E-Mail can be misunderstood to mean one of the following:
- Add an email account to Delta Chat
- Create a new email account and add it to Delta Chat
- Send a new encrypted email to a Thunderbird user (it is expected that mainly a bit more technical users will use this feature, who know what an encrypted email is)

Therefore, I think that it should be called "Send unencrypted email".

Reply:
also "send unencrypted email" doesn't really reflect what the option actually does, I already recommended calling it "new email thread" another even clearer option would be "new unencrypted group" because what the option does is not to create an email but a group/thread, btw actually in such groups it is clearer what to do and we could display subjects and show a subject field together with the message field

Reply:
there are at least as many opinions as ppl on the word "e-mail" as well as the "envelope" :) see discussions at  https://github.com/deltachat/deltachat-ios/pull/2770, https://mastodon.social/@devSJR@fosstodon.org/114845881860226371 or internal chats or pros and cons

let's just stay with the simple thing for now and not open another discussion here :)

Reply:
about the mastodon user, I think that is totally overthinking, people that will see the email envelope is people using classic email and receiving classic email from people and newsletters/services they know are not even using Delta Chat, also it is never mixed between encrypted message as that person seems to misunderstand, so there is no risk that someone would think that messages are more secure than the ones without the symbol

Reply:
(and I totally agree with not opening discussion here, I asked for review not to question the decisions that weren't taken by me I just did the same as iOS PR)